ruggedpod_api/server.py

```diff
@@ -64,7 +64,8 @@ def authenticate():
     token, expires = auth.get_token(request.args['username'],
                                     request.args['password'])
     response = make_response('', 201)
-    response.set_cookie('X-Auth-Username', request.args['username'], expires=expires)
+    response.set_cookie('X-Auth-Username', request.args['username'],
+                        expires=expires)
```

Comment:
Actually the pep8 configuration in the project allows up to [120 characters](https://github.com/RuggedPOD/ruggedpod-api/blob/ec3c1e122a2e32a73bb72f5c25806ffe6d2b68af/setup.cfg#L17) for a line. In the other hand, the travis build runs the pep8 checks and thus the master branch code style is always good regarding to the project rules.